Source/drlg_l4.cpp

```diff
@@ -1003,12 +1003,7 @@ static void uShape()
 			}
 			if (dung[i][j] == 1) {
 				// BUGFIX: check that i + 1 < 20 and j + 1 < 20 (fixed)

```

Comment:
Are these BUGFIX comments still relevant after your changes?

Reply:
They were helpful in avoiding a revert after merging with upstream when there was a big refactor. But this is becoming much less relevant and I will probably strip them after the next merge.